includes/admin/class-wcmp-export-consignments.php

```diff
@@ -129,7 +129,7 @@ public function setCustomItems(): void
             $product = $item->get_product();
             $country = $this->getCountryOfOrigin($product);
 
-            if (! empty($product)) {
+            if (!empty($product)) {
```

Comment:
```suggestion
            if (! empty($product)) {
```
zie de [php standaarden](https://wiki.dmp.zone/display/MD/DSR-1%3A+PHP+Standards#DSR-1:PHPStandards-Whitespacebetweenstatements)